challenge.md

```diff
@@ -6,19 +6,27 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+  git clone SSH URL.
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+  git diff HEAD^1 HEAD^2
```

Comment:
`git diff HEAD` would work best, and `git status` is also an acceptable answer, as it lists the files that have been altered since the last commit.

`HEAD^1` refer to the parent (previous) commit of the most recent one. `HEAD^2` is an invalid symbol unless the most recent commit has at least two parent commits. This is advanced git usage that I don't believe we covered in class.